bin/oref0-setup.sh

```diff
@@ -232,6 +232,13 @@ if [[ -z "$DIR" || -z "$serial" ]]; then
             echo
         fi
     fi
+    read -p "Would you like to [D]ownload precompiled go pump communication library or build them from [S]ource? [D]/S " -r
```

Comment:
The Go language should be capitalized any time we’re not referring to an actual lowercase command / file.